docusaurus/docs/learn/quickstarts/network/help/upgrade-quickstart-network.md

```diff
@@ -0,0 +1,175 @@
+Ziti releases can be found [here](https://github.com/openziti/ziti/releases). There is a helpful Ziti CLI script
+function called `getZiti` that will obtain the binary and will determine the correct architecture and OS for you.
+
+Here is the process for using `getZiti` to obtain the latest binary. You may optionally add `yes` as an argument to
+the `getZiti` function to have the new ziti binary automatically added to your $PATH
+
+```
+source /dev/stdin <<< "$(wget -qO- https://get.openziti.io/quick/ziti-cli-functions.sh)"
+getZiti
+```
+
+:::note
+After upgrading the Ziti binary, be sure to check your PATH to ensure it is pointing to the new binary.
+:::
+
+## Update Services
+
+If you have services to start up the network, update them to use the new binary file.
+
+For example, the `ExecStart` section will point to the full path to the old Ziti binary. Update this line to point to
+the path of the newly downloaded Ziti binary.
+
+<Tabs groupId="update-services">
+<TabItem value="28.2" label="v0.28.2 or Later">
+
+The `ExecStart` section will point to the full path to the old Ziti binary. Update this line to point to
+the path of the newly downloaded Ziti binary.
+
+For example, the following portion of the `ExecStart`
+
+```
+.../ziti-v0.28.0/ziti" controller run 
+```
+
+will be updated to the following (assuming a new version of `v0.30.4`).
+
+```
+.../ziti-v0.30.4/ziti" controller run
+```
+
+</TabItem>
+<TabItem value="pre28.2" label="v0.28.1 or Earlier">
+
+In version `v0.28.2`, `ziti-controller`, `ziti-router`, and `ziti-tunnel` were bundled into the `ziti` binary.
+Therefore, in these earlier versions, the `ExecStart` path will not only point to the older version of the Ziti binary
+but the executable used will be `ziti-router` and `ziti-controller`. When upgrading, it is important to change the path
+of the `ExecStart` section to use the new binary (`ziti`).
+
+For example, the following portion of the `ExecStart`
+
+```
+.../ziti-v0.27.0/ziti-controller" run 
+```
+
+will be updated to the following (assuming a new version of `v0.30.4`). Notice the version and executable change.
+
+```
+.../ziti-v0.30.4/ziti" controller run
+```
+
+</TabItem>
+</Tabs>
+
+## Reload and Start Ziti Services
+
+To reflect changes made to the Ziti services, reload the systemd daemon
+
+```
+sudo systemctl daemon-reload
+```
+
+Start Ziti services
+
+```
+sudo systemctl start ziti-controller ziti-router
+```
+
+</TabItem>
+<TabItem value="docker-quickstart" label="Docker Quickstart">
+
+## Docker Users
+
+For Docker users, the upgrade process is much simpler, all the files used for the network are stored in the
+`/persistent` directory of the docker containers. You may back up the files if desired, however the upgrade process
+follows these few instructions;
+
+1. Backup files in `/persistent` if desired
+1. Stop the docker containers but **do not** remove the volume.
+    1. For docker-compose users, use `docker compose down` (do not use the `-v` option)
```

Comment:
i would remove this. just saying "stop the containers" is enough for me. not to mention WHO KNOWS how they started it... :)